realm/realm-library/src/objectServer/java/io/realm/RealmAppConfiguration.java

```diff
@@ -175,10 +184,25 @@ public File getSyncRootDirectory() {
         return syncRootDir;
     }
 
+    // FIXME Doc
+    public CodecRegistry getCodecRegistry() { return codecRegistry; }
```

Comment:
Should probably be `getDefaultCodecRegistry`

Reply:
👍